Sources/Core/JavaModelRenderer.swift

```diff
@@ -308,6 +308,11 @@ public struct JavaModelRenderer: JavaFileRenderer {
             [JavaIR.Root.packages(names: [$0])]
         } ?? []
 
+        let nullabilityAnnotationType = JavaNullabilityAnnotationType(rawValue: params[.javaNullabilityAnnotationType] ?? "android-support")
```

Comment:
Can we use a `guard let` pattern here? `guard` prevents execution of the logic below if the `let` assignment fails. This would replace the `nullabilityAnnotationType == nil` check.

```
guard let nullabilityAnnotationType = JavaNullabilityAnnotationType(rawValue: params[.javaNullabilityAnnotationType] ?? "android-support") else {
   fatalError("java_nullability_annotation_type must be either android-support or androidx")
}
```

Additionally it would be good to include the value the user provided for `java_nullability_annotation_type` in this error message